src/application/agora/widgets/proposal.tsx

```diff
@@ -0,0 +1,106 @@
+import { classes } from 'shared/ui/utils';
+import { Chip, WidgetBase } from 'shared/ui/components';
+import {
+  getDifferenceInDays,
+  getEndsInLabel,
+} from 'shared/ui/utils/date-utils';
+
+import { ProposalData } from '../types';
+import { getProposalUrl } from '../utils';
+import { NavigationButton } from '../components/navigation-button';
```

Comment:
it won't change anything but just to follow convention please make index.ts to re-export components